tnt/src/main/java/org/dcsa/conformance/standards/tnt/action/TntAction.java

```diff
@@ -81,7 +80,7 @@ private void updateCursorFromResponsePayload(ConformanceExchange exchange) {
       }
     }
 
-    if (!dsp.equals(updatedDsp)) dsp.set(updatedDsp);
+    if (!dsp.get().equals(updatedDsp)) dsp.set(updatedDsp);
```

Comment:
I think this change can be changed in other standards as well as I believe dsp.equals(updatedDsp) is what being done in other standards too.

Reply:
Yes, my IntelliJ gave this warning. Not sure if it was generated by SonarLint plugin, which I recommend to install, or if it was by IntelliJ itself. Perhaps update your IDE, to fetch the latest enhancements, but this feature was already in for a longer period. 

I will search this in other code as well, and apply the fix. I think I already did at some places in the past PRs btw.